salt/modules/pcs.py

```diff
@@ -26,6 +26,23 @@ def __virtual__():
     return (False, "Missing dependency: pcs")
 
 
+def __use_new_commands():
+    """
+    The command line arguments of pcs changed after version 0.10
+    This will return True if the new arguments are needed and
+    false if the ole ones are needed
+    """
+    pcs_version = __salt__["pkg.version"]("pcs")
+    log.debug("PCS package version %s", pcs_version)
+
+    if __salt__["pkg.version_cmp"](pcs_version, "0.10") == 1:
+        log.info("New version, new rules")
```

Comment:
Please make these logs at the `debug` level.

Reply:
sorted all changes